core/aggregations/src/main/java/com/microsoft/gctoolkit/aggregators/RuntimeAggregator.java

```diff
@@ -0,0 +1,31 @@
+package com.microsoft.gctoolkit.aggregators;
+
+import com.microsoft.gctoolkit.aggregations.RuntimeAggregation;
+import com.microsoft.gctoolkit.aggregator.Aggregator;
+import com.microsoft.gctoolkit.event.jvm.JVMEvent;
+import com.microsoft.gctoolkit.event.jvm.JVMTermination;
+
+/**
+ * An Aggregator that collects only the DateTimeStamp and duration of a JVMEvent.
+ */
+public class RuntimeAggregator extends Aggregator<RuntimeAggregation> {
+
+    protected RuntimeAggregator(RuntimeAggregation aggregation) {
+        super(aggregation);
+    }
+
+    // This is atypical of an Aggregator. The typical pattern is to register the method that processes
```

Comment:
I'm trying to understand why this code if following a different pattern than the norm established by registering the callback.